docs/src/whatsnew/3.1.rst

```diff
@@ -16,6 +16,10 @@ This document explains the changes made to Iris for this release
 
    The highlights for this minor release of Iris include:
 
+   * Much faster import time, from minimised loading of submodules.
+     Note however that this may break existing code which does not declare full imports.
+     See :ref:`Minimised Imports <minimised_imports>`.
+   * Speedup for loading single phenomena from netcdf.  See :ref:`netcdf load speedup <fast_nc_load>`.
```

Comment:
@pp-mo The reference on line +115 requires to be indented i.e., it shouldn't have leading spaces, see `.. _fast_nc_load:`

Reply:
If you **_don't_** indent it, it breaks the numbering of the numbered list.
I found this as a fix from a stack overflow search.